src/main/cpp/geo/Profile.cpp

```diff
@@ -0,0 +1,38 @@
+#include <fstream>
+#include <boost/tokenizer.hpp>
+#include "Profile.h"
+#include "util/StringUtils.h"
+
+namespace stride {
+namespace geo {
+
+ProfileRef Profile::Parse(std::ifstream& csv_file)
+{
+	std::vector<City> cities;
```

Comment:
Hm, `clang-format` left this bit of the code untouched. Is it brace style that’s unusual? Because this is how it’s set up in our configuration file:
```
BraceWrapping:
  ...
  AfterFunction:   true
  ...
```